securedrop/source_templates/index.html

```diff
@@ -62,7 +62,7 @@
     Included here so the images can preload while the user is first
     reading the page. Hidden by default. -->
     <div class="bubble">
-      <p>You appear to be using the Tor Browser. You can turn the Security Slider to High in 4 easy steps!</p>
+      <p>You appear to be using the Tor Browser. In Tor's "Security Settings" pane (accessible from the toolbar's onion-button), we reccommend turning Security Slider to High—it's easy-peasy!</p>
```

Comment:
@redshiftzero also, I'm not clear on what you changed or want me to change... there's sooo much going on in this PR, now (and I kinda intended it to target 0.3.11, was that bad?). The use of -> and -> and -> is easy for geeks to figure-out, but keeping the written directions as clear sentences, will go farther with both speaking clearly to users, and keeping a cadence in the tone of the writing. Smoothly cadenced writing = less anxiety. :)

Separately: I saw you added an error-bang into the "please quit Tor" end-message, and that it's red. While that message is unlikely to dissuade users from using the product, the user never did anything "wrong," so that's a poor use of both the icon and the color—hence, the purple bubble w/ the fingerprint recommendation. Dialogs also shouldn't have numbered-points in them. It's just not a convention, and is odd. Are the fellas just trying to bang-out that release w/o the recommended dialogs, or did y'all want more time to iterate on them?